src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -345,10 +358,12 @@ $pf-c-toolbar--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl"
     }
   }
 
+  // stylelint-disable max-line-length
```

Comment:
You shouldn't need these - this rule was deprecated https://stylelint.io/user-guide/rules/max-line-length/

As long as you have the latest from v5 in your PR, it shouldn't error. If they're still showing up in vscode, you may need to restart vscode as I've noticed it doesn't always read new stylelint rules properly.